dev-libs/libcgroup/files/cgconfig.initd-r1

```diff
@@ -1,16 +1,17 @@
 #!/sbin/openrc-run
```

Comment:
Please copy these (confd+initd) into a new file, or we're going to change the image of 3.0.0 inadvertently